BootloaderCorePkg/Library/AcpiInitLib/AcpiInitLib.c

```diff
@@ -380,6 +383,81 @@ UpdateMadt (
   return EFI_SUCCESS;
 }
 
+/**
+  This function updates FWST ACPI data structure based on ESRT table.
+
+  @param[in]  Current               Pointer to MADT table
+  @retval     EFI_SUCCESS           Patched the MADT
+              EFI_UNSUPPORTED       Sysinfo table not found.
+              EFI_OUT_OF_RESOURCES  If less MADT entries
```

Comment:
should ESRT and MADT be something else (FWST)?

Reply:
corrected the comments in the code